staging/src/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go

```diff
@@ -40,6 +40,12 @@ const (
 	//
 	// CustomResourceSubresources defines the subresources for CustomResources
 	CustomResourceSubresources utilfeature.Feature = "CustomResourceSubresources"
+
+	// owner: @mbohlool
+	// alpha: v1.12
+	//
+	// CustomResourceWebhookConversion defines the webhook conversion for Custom Resources.
+	CustomResourceWebhookConversion utilfeature.Feature = "CustomResourceWebhookConversion"
```

Comment:
You also need to add `CustomResourceWebhookConversion` to `defaultKubernetesFeatureGates` in this file

Reply:
Yeah, it is unfortunate that we have the list of defaults duplicated for apiextensions. Done.